build-system/externs/amp.extern.js

```diff
@@ -80,6 +80,25 @@ Element.prototype.__AMP_SHADOW_ROOT;
 /** @type {?ShadowRoot} */
 Element.prototype.shadowRoot;
 
+// Fullscreen methods
+Document.prototype.cancelFullScreen = function () {};
```

Comment:
Do we need to assign these function values here?

Reply:
no we don't, although I figure it is slightly better than `?`

Reply:
I removed the function